cmd/controller-manager/app/controller-manager.go

```diff
@@ -303,7 +313,18 @@ func setOptionsByFederationConfig(opts *options.Options) {
 	if fedConfig == nil {
 		// FederationConfig could not be sourced from --federation-config or from the API.
 		// create a new `FederationConfig` with default values.
-		fedConfig = &corev1a1.FederationConfig{}
+		fedConfig = &corev1a1.FederationConfig{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      util.FederationConfigName,
+				Namespace: opts.Config.FederationNamespace,
+			},
+		}
+
+		qualifiedName := util.QualifiedName{
+			Namespace: fedConfig.Namespace,
+			Name:      fedConfig.Name,
+		}
+		glog.Infof("Creating FederationConfig %q with default values", qualifiedName)
```

Comment:
In addition to providing helpful output at runtime, log statements can serve as informal comments when reading the code.  In the latter capacity they work best when they precede the action they are documenting, e.g.

```golang
// Comment precedes action

doSomething()
// It's time to do something

vs

// It's time to do something
doSomething()

// Logging precedes action

doSomething()
glog.Log("It's time to do something")

vs

glog.Log("It's time to do something")
doSomething()
```